packages/components/psammead-radio-schedule/src/index.jsx

```diff
@@ -16,6 +17,9 @@ const StartTimeWrapper = styled.div`
 const StyledGrid = styled(Grid)`
   padding: 0;
```

Comment:
Looks good - another instance where previously with `styled-components`, a style outside of a media query was successfully being applied across all device sizes. Now, with emotion, we need to explicitly overwrite the style within a media query.